package.json

```diff
@@ -10,7 +10,8 @@
     "build": "ember build",
     "start": "ember server",
     "coverage": "COVERAGE=true ember test",
-    "test": "node_modules/.bin/semistandard blueprints/**/*.js tests/**/*.js addon/**/*.js app/**/*.js && npm run coverage",
+    "test": "semistandard blueprints/**/*.js tests/**/*.js addon/**/*.js app/**/*.js && npm run coverage",
```

Comment:
this is using local libraries, it is unnecessary to to preface with node_modules/.bin it already does that